model/metrics/messaging-metrics.yaml

```diff
@@ -0,0 +1,80 @@
+groups:
+  - id: metric.messaging.attributes
+    type: attribute_group
+    stability: experimental
+    brief: "Common messaging metrics attributes."
+    extends: attributes.messaging.common.minimal
+    attributes:
+      - ref: messaging.system
+        requirement_level: required
+      - ref: messaging.destination.partition.id
+      - ref: messaging.destination.name
+        requirement_level:
+          conditionally_required: if and only if `messaging.destination.name` is known to have low cardinality. Otherwise, `messaging.destination.template` MAY be populated.
+      - ref: messaging.destination.template
+        requirement_level:
+          conditionally_required: if available.
+
+  # durations
+  - id: metric.messaging.client.request.duration
+    type: metric
+    metric_name: messaging.client.request.duration
```

Comment:
naming it "request" seems like a good choice over "operation" in order to help mitigate the confusion around whether this also includes "process" times (which have "messaging.operation.name" and so seem like they should be included in anything called "operation")